docs/scanner-pools.md

```diff
@@ -0,0 +1,182 @@
+# Manage Your Scanner Pools
+
+## Introduction
+
+A scanner pool contains a group of nodes that scan a specific chain. Each pool belongs to a specific owner and is minted as an NFT (ERC-721) upon registration. These pools require the operators to deposit a total amount of stake which covers the minimum-per-scan-node amount for each node.
+
+When registering any node to a pool, make sure to stake this minimum amount of **2500 FORT** on the pool first.
+
+In addition, the scanner pools allow other FORT token holders (delegators) to stake on your pool and earn rewards. As the pool operator, you can decide
+
+- how much of the delegator or operator stake to be allocated,
+- how much commission you want from delegators' rewards.
+
+## Creating a new scanner pool
+- In the Forta App navigate to “My Node Pools”
+
+![My Node Pools](MyNodePoolsDropdown.png)
+
+- Click “Add Scanner Pool”
+![My Node Pools Tab](MyNodePoolsTab.png)
+
+- Select a chain for your node pool to monitor and register your node pool
+
+![Create Node Pool](CreateNodePool.png)
+
+- Accept the transaction for creating your node pool and wait for it to be confirmed
+
+## Depositing stake
+
+### Depositing on your own pool
+- In the Forta App navigate to “My Node Pools”
+
+![My Node Pools](MyNodePoolsDropdown.png)
+
+- Click on node pool you would like to add stake
+
+![My Node Pools Tab](MyNodePoolsTab.png)
```

Comment:
the image used here doesnt actually display any node pools. can we update it?

Reply:
I think it shows the "My Node Pools" tab correctly. It displays the state when any operator starts from scratch, doesn't have any pools and needs to start by creating one.